dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityOptions.nlsprops

```diff
@@ -172,6 +172,11 @@ sslCert.option-desc.sigAlg=\
 \tSignature algorithm of the certificate.\n\
 \tThe default signature algorithm is {8}.
 
+sslCert.option-key.createJKSFile=\
+\ \ \ \ --createJKSFile
+sslCert.option-desc.createJKSFile=\
+\tOptional.  Unless specified and set to true, a PKCS12 file will be generated.\n\
+\tOtherwise, a JKS file will be generated.
```

Comment:
Change
"Unless specified and set to true, a PKCS12 file **will be** generated. Otherwise, a JKS file **will be** generated."
to
"Unless specified and set to true, a PKCS12 file **is** generated. Otherwise, a JKS file **is** generated."

Reply:
@ManasiGandhi , Manasi, the wording has completely changed.  Please re-review